explore/src/main/scala/explore/targeteditor/AsterismEditor.scala

```diff
@@ -154,26 +167,57 @@ object AsterismEditor {
                 val optional =
                   Optional[List[TargetWithId], Target](_.find(_.id === targetId).map(_.target))(
                     target =>
-                      _.map(twid =>
+                      _.map { twid =>
+                        println(s"targetId: $targetId selectedTargetId: ${selectedTargetId.get}")
```

Comment:
println?

Reply:
Yeah. That's a special production ready `println`. 😬 I'll get rid of it.